modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeNumberInput.ts

```diff
@@ -37,9 +38,14 @@ const createBespokeNumberInput = (editor: Editor, backstage: UiFactoryBackstage,
 
   const changeValue = (f: (v: number, step: number) => number, fromInput: boolean, focusBack: boolean): void => {
     const text = getValueFromCurrentComp(currentComp);
-    const parsedText = Dimension.parse(text, [ 'unsupportedLength' ]);
+    const parsedText = Dimension.parse(text, [ 'unsupportedLength', 'empty' ]);
     const value = parsedText.map((res) => res.value).getOr(0);
-    const unit = parsedText.map((res) => res.unit).getOr('');
+    const defaultUnit = getFontSizeInputDefaultUnit(editor);
```

Comment:
Import this a named import so that it becomes `Options.getFontSizeInputDefaultUnit` then it clear that it's a user option that is getting pulled in.

Reply:
done